pika/channel.py

```diff
@@ -282,27 +274,27 @@ def basic_consume(self, consumer_callback,
         http://www.rabbitmq.com/confirms.html
         http://www.rabbitmq.com/amqp-0-9-1-reference.html#basic.consume
 
-
-        :param callable consumer_callback: The function to call when consuming
-            with the signature consumer_callback(channel, method, properties,
-                                                 body), where
+        :param queue: The queue to consume from. Use the empty string to specify the
+                      most recent server-named queue for this channel.
```

Comment:
The last sentence about empty string is a proprietary feature of RabbitMQ, right? If so, need to make that clear or remove the sentence.

Reply:
Are you planning to remove that sentence or add a reference to RabbitMQ? Ditto in `basic_get` and wherever else this is mentioned.

Reply:
Sorry, never mind - it appears that this is a feature of the AMQP 0.9.1 specification, so not specific to RabbitMQ